etc/subsystem_manifest/main.go

```diff
@@ -39,23 +36,26 @@ type substemManifest struct {
 	Subsystem  string              `json:"subsystem"`
 	Version    string              `json:"version"`
 	Platform   string              `json:"platform"`
-	ObjectPath string              `json:"object-path"`
+	ObjectPath string              `json:"upload-path"`
```

Comment:
[minor] json (and bson) tags should match the actual struct tag to avoid confusion. E.g. ObjectPath should be UploadPath.